db/migrate/20230921021601_add_user_to_replies.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToReplies < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :replies, :user, null: false, foreign_key: true
```

Comment:
same here replies already has the user_id column, this result in `PG::DuplicateColumn: ERROR: `